resources/views/settings/alerts.blade.php

```diff
@@ -43,7 +43,7 @@
                         <!-- Alerts Enabled -->
                         <div class="form-group {{ $errors->has('alert_email') ? 'error' : '' }}">
```

Comment:
Can you change this class to `alerts_enabled` too? I'll merge the PR then (don't bother redirecting to develop - this was done off of master, so it's just a PITA for a PR this small.)

Reply:
I will make sure to branch from develop for sure. My bad 👍

Reply:
No worries - I’d just much rather you get the PR credit :)